src/ports/postgres/modules/graph/graph_utils.py_in

```diff
@@ -27,104 +27,103 @@
+        **locals()))
+
+    _assert(edge_table and edge_table.strip().lower() not in ('null', ''),
+            "Graph {func_name}: Invalid edge table name!".format(**locals()))
+    _assert(table_exists(edge_table),
+            "Graph {func_name}: Edge table ({edge_table}) is missing!".format(
+        **locals()))
+    _assert(not table_is_empty(edge_table),
+            "Graph {func_name}: Edge table ({edge_table}) is empty!".format(
+        **locals()))
+
+    existing_cols = set(unquote_ident(i) for i in get_cols(vertex_table))
+    _assert(vertex_id in existing_cols,
+            """Graph {func_name}: The vertex column {vertex_id} is not present in vertex table ({vertex_table}) """.
+            format(**locals()))
+    _assert(columns_exist_in_table(edge_table, edge_params.values()),
+            """Graph {func_name}: Not all columns from {cols} are present in edge table ({edge_table})""".
+            format(cols=edge_params.values(), **locals()))
+
+    return None
+
 
 def get_graph_usage(schema_madlib, func_name, other_text):
 
-	usage = """
-----------------------------------------------------------------------------
-                            USAGE
-----------------------------------------------------------------------------
- SELECT {schema_madlib}.{func_name}(
-    vertex_table  TEXT, -- Name of the table that contains the vertex data.
-    vertex_id     TEXT, -- Name of the column containing the vertex ids.
-    edge_table    TEXT, -- Name of the table that contains the edge data.
-    edge_args     TEXT{comma} -- A comma-delimited string containing multiple
-                        -- named arguments of the form "name=value".
-    {other_text}
-);
-
-The following parameters are supported for edge table arguments ('edge_args'
-	above):
-
-src (default = 'src')		: Name of the column containing the source
-				vertex ids in the edge table.
-dest (default = 'dest')		: Name of the column containing the destination
-				vertex ids in the edge table.
-weight (default = 'weight')	: Name of the column containing the weight of
-				edges in the edge table.
-""".format(schema_madlib=schema_madlib, func_name=func_name,
-	other_text=other_text, comma = ',' if other_text is not None else ' ')
-
-	return usage
+    usage = """
+    ----------------------------------------------------------------------------
+                                USAGE
+    ----------------------------------------------------------------------------
+     SELECT {schema_madlib}.{func_name}(
+        vertex_table  TEXT, -- Name of the table that contains the vertex data.
+        vertex_id     TEXT, -- Name of the column containing the vertex ids.
+        edge_table    TEXT, -- Name of the table that contains the edge data.
+        edge_args     TEXT{comma} -- A comma-delimited string containing multiple
+                            -- named arguments of the form "name=value".
+        {other_text}
```

Comment:
`other_text` is also a string that is passed to this function. Shouldn't that string also be indented with a tab's space? For instance, this is how help on `sssp` looks like now:
```
                                    graph_sssp
----------------------------------------------------------------------------------
                                                                                 
 Given a graph and a source vertex, single source shortest path (SSSP)           
 algorithm finds a path for every vertex such that the sum of the                
 weights of its constituent edges is minimized.                                  
                                                                                 
                                                                                 
     ----------------------------------------------------------------------------
                                 USAGE                                           
     ----------------------------------------------------------------------------
      SELECT madlib.graph_sssp(                                                  
         vertex_table  TEXT, -- Name of the table that contains the vertex data. 
         vertex_id     TEXT, -- Name of the column containing the vertex ids.    
         edge_table    TEXT, -- Name of the table that contains the edge data.   
         edge_args     TEXT, -- A comma-delimited string containing multiple     
                             -- named arguments of the form "name=value".       
         source_vertex INT,  -- The source vertex id for the algorithm to start. 
     out_table     TEXT, -- Name of the table to store the result of SSSP.    
     grouping_cols TEXT  -- The list of grouping columns.                        
     );                                                                          
```
the description of `out_table` and `grouping_cols` comes from the `other_text` param.